addons/mail/static/src/models/composer/composer.js

```diff
@@ -540,10 +551,17 @@ function factory(dependencies) {
         attachments: many2many('mail.attachment', {
             inverse: 'composers',
         }),
+        /**
+         * Temporary flag for all composer attachments.
+         */
+        attachmentsAreTemporary: attr({
```

Comment:
This field watches the uploading (= temporary) status of attachments linked to this composer.
Useful to determine whether there are some attachments that are being uploaded.

_(1st sentence => "what", 2nd sentence => "why")_